src/components/Accordion/index.js

```diff
@@ -9,10 +14,23 @@ export default class Accordion extends Component {
     constructor(props) {
         super(props);
         this.handleToggleSection = this.handleToggleSection.bind(this);
+
+        this.containerRef = React.createRef();
+        this.registerAccordionSection = this.registerAccordionSection.bind(this);
+        this.unregisterAccordionSection = this.unregisterAccordionSection.bind(this);
+        this.handleKeyPressed = this.handleKeyPressed.bind(this);
+        this.keyHandlerMap = {
+            [RIGHT_KEY]: () => this.selectAccordionSection(RIGHT_SIDE),
+            [LEFT_KEY]: () => this.selectAccordionSection(LEFT_SIDE),
+            [DOWN_KEY]: () => this.selectAccordionSection(RIGHT_SIDE),
+            [UP_KEY]: () => this.selectAccordionSection(LEFT_SIDE),
+        };
+
         this.state = {
             activeNames: props.activeSectionNames,
             multiple: props.multiple,
             privateOnToggleSection: this.handleToggleSection,
+            accordionSection: [],
```

Comment:
accordionSection? This is an accordionSection?
childrenRegistered or accordionChildren